client/token_claim_create.go

```diff
@@ -73,26 +73,26 @@ func (cli *VanClient) TokenClaimCreate(ctx context.Context, name string, passwor
 	}
 	host := fmt.Sprintf("%s.%s", types.ControllerServiceName, cli.Namespace)
 	localOnly := true
-	if cli.RouteClient != nil {
-		route, err := cli.RouteClient.Routes(cli.Namespace).Get(types.ClaimRedemptionRouteName, metav1.GetOptions{})
```

Comment:
Another approach - the one used for creating cert tokens - is to check the error and to treat IsNotFound as implying routes are not in use.

Reply:
I like that approach and will make it similar here as well.